CondFormats/SerializationHelper/README.md

```diff
@@ -0,0 +1,61 @@
+# Overview of CondFormats/SerializatonHelper
+
+This package provides a mechanism to load plugins which are capable of doing C++ object serialization/deserialization
+using the same mechanism (boost serialization) as the conditions database.
+
+
+## Using the Code
+The plugin factory is named `SerializationHelperFactory` which returns instances of `SerializationHelperBase`. The
+latter provides methods to serialize an C++ class to a buffer and to deserialize from a buffer to an instance of a C++
+class. Polymorphism is supported for serialization/deserialization.
+
+## Registering a Plugin
+
+The normal way to register a plugin is to just register a plugin for the CondDBESSource. The registration macros for
+the CondDBESSource contain calls to the macros used to generate the SerializationHelpers. One caveat is for the case
+where the same C++ class is begin stored in multiple Records within CondDBESSource. As the CondDBESSource registration
+macro includes both the Record name and the C++ class name, but the SerializationHelper system only cares about the
+C++ class name, this situation can cause the registration of the same C++ class multiple times which leads to a
+compilation error. This problem can be overcome by replacing one of the `REGISTER_PLUGIN` calls with `REGISTER_PLUGIN_NO_SERIAL`. E.g.
```

Comment:
It's not fully clear to me if these compilation errors you mention here are occurring already now (I'm not aware of them but it might just be my ignorance), or they would occur after the merge of this PR, hence the need to add the `REGISTER_PLUGIN_NO_SERIAL` macro?